docs/getting-started-tutorial/5-basic-forms.adoc

```diff
@@ -0,0 +1,106 @@
+:idprefix:
+:idseparator: -
+:toc: right
+:toclevels: 2
+
+# Basic Forms
+
+It's a recurring and important theme that Lift is secure by default. This
+manifests in the way that forms are constructed as well. Form fields in Lift
+are associated with a callback function that runs when the field is submitted
+with the form. On the client, the field name is always unique to this page
+load, and this unique field name is a cryptographically secure random value
+that is associated to the callback function you specify on the server
+footnote:[What about using your own field names, you may ask? You can always do
+that. To access a submitted field with a given name, you can use
+`S.param("field name")`. You'll get back a `Box` that will be `Full` and
+contain the value you submitted if it was submitted. The `Box` will be `Empty`
+if the field wasn't submitted in this request. However, be very careful about
+using this method, since it exposes you to CSRF attacks.]. This makes Lift
```

Comment:
Should there be a link to the Lift Cookbook entry to [Plain Old Form Processing](http://chimera.labs.oreilly.com/books/1234000000030/ch03.html#_solution_24) here?

Reply:
I guess we'll need to figure out the Lift Cookbook situation… So will punt here.